src/App/pages/RecipeDetails/RecipeDetails.tsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import { NavLink, useParams } from 'react-router-dom';
+import { getRecipeById } from '../../../services/recipesService';
+import styles from './RecipeDetails.module.scss';
+import { RecipeDetails as RecipeDetailsType } from 'types/recipe';
+import { getTestRecipe } from 'utils/getTestRecipe';
+import { IRawSingleRecipeData } from 'utils/types';
+import ArrowDownIcon from 'components/icons/ArrowDownIcon';
+import arrowRound from 'assets/arrow-round.svg';
```

Comment:
увидела svgr👍 плагин можно использовать так:
```suggestion
import ArrowRound from 'assets/arrow-round.svg?react';
```

и ниже в коде можно будет заменить `img` на `<ArrowRound />`